### PR TITLE
E1708 Improvements to staggered-deadline assignments

### PR DESCRIPTION
**Pull request by SmokingSadhus**

### CODE REVIEW

File: db/migrate/20161202124659_add_review_last_graded_date_to_participants.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddReviewLastGradedDateToParticipants < ActiveRecord::Migration
+  def change
+    add_column :participants, :review_last_graded_date, :datetime
```
Comment (yangsong8) [inline db/migrate/20161202124659_add_review_last_graded_date_to_participants.rb]: should b called review_graded_at

**Comment by Winbobob** [comment]
- I don't think add `review_graded_at` to `participants` table is correct. For newly-added reviews in staggered-deadline assignment, we can check the timestamp of responses.
 - I don't think add a new column in assignment editting page, due date tab is the right way to meet the 2nd requirement. I think what professor means is for topic deadlines instead of assignment deadlines.

**Comment by Winbobob** [comment]
See dc9a90be9248900d8f70b013f6524ca669df9ed4

**Comment by Winbobob** [comment]
See 93897ffe6a05d5369557c174300f6f1a6beb71c0

**Pull request closed without merging**